lib/environment/EnvironmentsController.js

```diff
@@ -55,10 +55,19 @@ class EnvironmentsController extends BaseController {
         });
       },
       (next) => {
-        const newEnv = {
-          name: options.name
-        };
-        this.environmentsService.create(newEnv, appId, next);
+        if (options.file) {
+          const info = {
+            [AppOptionsName.APP]: appId,
+            file: options.file,
+            name: options.name
+          };
+          this.cliManager.processConfigFile(OperationType.CREATE, info, next);
```

Comment:
The "info" variable is called "options" in the processConfigFile method. I see that you already have "options" here, but you should probably call this "processOptions" or something like that. "info" does not have any meaning.